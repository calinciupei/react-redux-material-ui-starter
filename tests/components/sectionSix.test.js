import React from 'react';
import { shallow } from 'enzyme';

import SectionSix from '../../src/components/sectionSix/sectionSix.component';

let context;

const createComponent = () => shallow(<SectionSix />);

describe('<SectionSix />', () => {
  context = createComponent();

  it('should render correctly', () => {
    expect(context.dive()).toMatchSnapshot();
  });
});
