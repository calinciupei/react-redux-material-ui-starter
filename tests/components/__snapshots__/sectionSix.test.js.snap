// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionSix /> should render correctly 1`] = `
<WithStyles(Grid)
  alignItems="center"
  className="SectionSix-section-1"
  container={true}
  direction="row"
  justify="space-between"
>
  <WithStyles(Grid)
    className="SectionSix-sectionHeader-2"
    item={true}
    xs={12}
  >
    <WithStyles(Typography)
      component="h3"
      variant="h3"
    >
      <b>
        Section Six
      </b>
    </WithStyles(Typography)>
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
